src/components/Footer/footer.js

```diff
@@ -1,41 +1,61 @@
 import { BrowserRouter as Router, NavLink } from "react-router-dom";
 import styles from "./footer.module.scss";
+import logo from "../../assets/image/footer/Layer 2.png";
 
 export const Footer = () => {
   return (
     <footer className={styles.footer}>
-      <img className={styles.logo} alt="logo" />
+      <div className={styles.logo_column}>
+        <img className={styles.logo} src={logo} alt="logo" />
+        <p className={styles.description}>
+          Wszystkie usługi dla Twojego pupila w jednym miejscu.
+        </p>
+        <p>Copyright &copy; 2022</p>
+      </div>
       <section className={styles.column}>
-        <header className={styles.header}>O nas</header>
+        <header className={styles.header}>Usługi dla zwierząt</header>
         <Router>
           <NavLink to="/" className={styles.link}>
```

Comment:
Use a `text-shadow` to simulate `bold` or change a bit styles cause now after hover it makes other elements move. It doesn't look good then.